lib/start-with-txns.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+export CONTRACTS_PATH=$(pwd)/out/config/splunk/contracts
```

Comment:
the contracts path is /out/config/contracts if you use the txgenerate tool only. If you include splunk then this folder does exist because it's copied along with the whole splunk folder

Reply:
this was removed